webrtc/modules/audio_device/win/audio_device_core_win.h

```diff
@@ -238,6 +240,9 @@ class AudioDeviceWindowsCore : public AudioDeviceGeneric
     void _Lock() { _critSect.Enter(); };
     void _UnLock() { _critSect.Leave(); };
 
+private:    // Loopback capture support
```

Comment:
@jstys92 : this is sort of the style of this file (having multiple `private` blocks with grouped functions)...